back-end/middlewares/authentication.js

```diff
@@ -3,8 +3,8 @@ const jwt = require('jsonwebtoken');
 const authentication = (req, res, next) => {
   try {
     const token = req.headers.authorization.split(' ')[1];
-    const decode = jwt.verify(token, 'secretValue');
-    req.user = decode;
+    const decode = jwt.verify(token, process.env.SECRET_KEY);
+    req.userEmail = decode.email;
```

Comment:
Are you sure that we need only email in req object?

Reply:
fix it